mitodl/micromasters
# Fixed seed data for naive timestamps (#2712)

#### What are the relevant tickets?
Fixes #2712 

#### What's this PR do?
Adds UTC offsets to seed data and test data.

#### How should this be manually tested?
- Run `unseed_db` to remove existing seed data
- Run `seed_db` and verify you don't see the output described in the ticket. You should only see 3 lines of stdout.
- Verify you can still see the seed data users in the learner search.

#### Where should the reviewer start?
programs.json

#### What GIF best describes this PR or how it makes you! feel?
![](https://media.giphy.com/media/26Do6la9cIiHvIwMM/giphy.gif)
